docs/pubsub_snippets.py

```diff
@@ -311,24 +311,21 @@ def subscription_pull(client, to_delete):
     subscription.create()
     to_delete.append(subscription)
 
-    # [START subscription_pull_none_pending]
-    pulled = subscription.pull(max_messages=1)
-    # [END subscription_pull_none_pending]
```

Comment:
I had intended to use it, but then ended up dropping it when I added the `max_messages=2` argument to the `subscription_pull` snippet below (the one which actually fixes #1756).